### PR TITLE
add submodule "anserini-eval"

### PR DESCRIPTION
**Pull request by richard3983**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "anserini-eval"]
+	path = anserini-eval
```
Comment (lintool) [inline .gitmodules:2]: path should be `eval/` for consistency across repos

Comment (richard3983) [reply .gitmodules]: I have updated it.

**Comment by lintool** [comment]
hi @richard3983 - I'll let @ronakice and @rodrigonogueira4 try it out and they can give you 👍

**Comment by ronakice** [comment]
LGTM, thanks for adding this!

**Pull request merged**